README.md

```diff
@@ -1,25 +1,82 @@
-# `corppa`  PPA full-text corpus utilities
+# corppa
+
+This repository is research software developed as part of the [Ends of Prosody](https://cdh.princeton.edu/projects/the-ends-of-prosody/), which is associated with the [Princeton Prosody Archive](https://prosody.princeton.edu/) (PPA). This software is particularly focused on research and work related to PPA full-text and page image corpora.
+
+> [!WARNING]
+> This code is primarily for internal team use. Some portions of it may eventually be useful for participants of the [Ends of Prosody conference](https://cdh.princeton.edu/events/the-ends-of-prosody/) or be adapted or used elsewhere.
+
+## Basic Usage
+
+### Installation
+
+Use pip to install as a python package directly from GitHub.  Use a branch or tag name, e.g. `@develop` or `@0.1` if you need to install a specific version.
+
+```sh
+pip install git+https://github.com/Princeton-CDH/ppa-nlp.git#egg=corppa
+```
+or
+```sh
+pip install git+https://github.com/Princeton-CDH/ppa-nlp.git@v0.1#egg=corppa
+```
+
```

Comment:
Anything to add about conda install?

(I didn't actually test these and probably should)